pkg/chains/signing_test.go

```diff
@@ -160,25 +159,31 @@ func TestTaskRunSigner_SignTaskRun(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			cleanup := setupMocks(tt.backends, &mockRekor{})
+			cleanup := setupMocks(&mockRekor{})
 			defer cleanup()
 
 			ctx, _ := rtesting.SetupFakeContext(t)
 			ps := fakepipelineclient.Get(ctx)
 
-			ctx = config.ToContext(ctx, &config.Config{
+			cfg := &config.Config{
 				Artifacts: config.ArtifactConfigs{
 					TaskRuns: config.Artifact{
 						Format:         "tekton",
 						StorageBackend: sets.NewString(tt.configuredBackend),
 						Signer:         "x509",
 					},
 				},
-			})
+			}
+			ctx = config.ToContext(ctx, cfg.DeepCopy())
 
+			logger := logging.FromContext(ctx)
+			SecretPath := "./signing/x509/testdata/"
```

Comment:
Typically local variables are lower case to avoid confusion with package level exported variables.
```suggestion
			secretPath := "./signing/x509/testdata/"
```